cmd/telemeter-client/main.go

```diff
@@ -292,9 +312,9 @@ func (o *Options) Run() error {
 
 	if len(o.Listen) > 0 {
 		handlers := http.NewServeMux()
-		telemeterhttp.AddDebug(handlers)
-		telemeterhttp.AddHealth(handlers)
-		telemeterhttp.AddMetrics(handlers)
+		telemeterhttp.DebugRoutes(handlers)
```

Comment:
as we discussed, we need to follow up and do some cleanup here

Reply:
yes :+1: